content/learn/quick-start/getting-started/setup.md

```diff
@@ -130,17 +130,34 @@ It's not uncommon for debug builds using the default configuration to take multi
 Fortunately, there is a simple fix, and we don't have to give up our fast iterative compiles! Add the following to your `Cargo.toml`:
 
 ```toml
-# Enable a small amount of optimization in debug mode
+# Enable a small amount of optimization in debug mode.
 [profile.dev]
 opt-level = 1
 
-# Enable high optimizations for dependencies (incl. Bevy), but not for our code:
+# Enable a large amount of optimization in debug mode for dependencies.
```

Comment:
I prefer the old one tbh

Reply:
Maybe. I wanted it to parallel the previous comment so the differences are obvious, and "including Bevy" should be trivially implied?

Reply:
If both of you prefer the new one, let's take that :)